test/unit/data/model/mapping/test_gxy_model_mapping.py

```diff
@@ -1317,6 +1317,84 @@ def test_relationships(self, session, cls_, group, role):
             assert stored_obj.role.id == role.id
 
 
```

Comment:
We are dropping all mapping tests. It's not merged yet only because it got stuck in a large SA2.0 PR which is not yet ready for review. There's a more detailed explanation in the backend meeting minutes from 1/31/23 (see  bullet point 1.3). So this file will be dropped. (sorry for any wasted time - I should've dropped those already)

Reply:
No worries! I had a vague feeling/memory this was the case but since the tests were already there in the original PR... I didn't think of it too much :man_shrugging: 

I'll drop them :+1: Thank you!